app/presenters/trashed_revision_presenter.rb

```diff
@@ -0,0 +1,73 @@
+class TrashedRevisionPresenter < BasePresenter
+  presents :revision
+
+  def created_at_ago
+    h.local_time_ago(revision.created_at)
+  end
+
+  def info
+    [
+      icon,
+      title,
+      type.downcase,
+      location,
+    ].join(" ").html_safe
+  end
+
+  private
+
+  def icon
+    # Set icon css depending on object type.
+    icon_css = %w{fa}
+    icon_css << case type
+                when 'Evidence'
+                  'fa-flag'
+                when 'Issue'
+                  'fa-bug'
+                when 'Note'
+                  'fa-file-text-o'
+                else
+                  ''
+                end
+    h.content_tag(:i, '', class: icon_css)
+  end
+
+  def location
+    # Get node if object is a Note or an Evidence.
+    if ['Note','Evidence'].include?(type)
+      if trashed_object.node
+        "at " + h.link_to(trashed_object.node.label, trashed_object.node)
+      else
+        'at a Node which has since been deleted'
+      end
+    else
+      ''
+    end
+  end
+
+  def title
+    # Get title or content first characters.
+    if trashed_object.fields.empty?
```

Comment:
here we're assuming that the trashed object has fields, in which case, using `HasFields#title` may just be a more consistent option.